net/https-dns-proxy/files/https_dns_proxy.init

```diff
@@ -1,39 +1,37 @@
 #!/bin/sh /etc/rc.common
 
 START=80
-
 USE_PROCD=1
-PROG=/usr/sbin/https_dns_proxy
-
-start_instance() {
-	local cfg="$1"
-	local listen_addr listen_port user group
 
-	config_get listen_addr "$cfg" listen_addr
-	config_get listen_port "$cfg" listen_port
-	config_get user "$cfg" user
-	config_get group "$cfg" group
-	config_get subnet_addr "$cfg" subnet_addr
-	config_get proxy_server "$cfg" proxy_server
-	config_get url_prefix "$cfg" url_prefix
+PROG=/usr/sbin/https_dns_proxy
 
-	if [ -n "$subnet_addr" ]; then
-		subnet_param="-e $subnet_addr"
-	fi
+xappend() { param="$param $1"; }
+
+append_parm() {
+	local section="$1"
+	local option="$2"
+	local switch="$3"
+	local default="$4"
+	local _loctmp
+	config_get _loctmp "$section" "$option" "$default"
+	[ -z "$_loctmp" ] && return 0
+	xappend "$switch $_loctmp"
+}
 
-	if [ -n "$proxy_server" ]; then
-		proxy_param="-t $proxy_server"
-	fi
+start_instance() {
+	local cfg="$1" param
 
-	if [ -z "$url_prefix" ]; then
-		url_prefix="https://dns.google.com/resolve?"
-	fi
+	append_parm "$cfg" 'listen_addr' '-a' '127.0.0.1'
+	append_parm "$cfg" 'listen_port' '-p' '5053'
+	append_parm "$cfg" 'bind_addr' '-b'
```

Comment:
In the interest of smaller configs, should these (-b and -r) be defaulted to one provider and config options commented out as examples or something? I have no preference as to which you use.

Reply:
Same argument as the above -- I'd rather leave these both in the config file than create a README.

Reply:
OK. Maybe then comment out the second config as an example in the config file to avoid running two daemons when most people will only ever use one?